pkg/scaffold/api.go

```diff
@@ -159,6 +160,11 @@ func (api *API) scaffoldV2() error {
 				Resource: r},
 			&resourcev2.Group{Resource: r},
 			&resourcev1.CRDSample{Resource: r},
+			// TODO(mengqiy): figure out how we should deal with the kustomization.yaml when the user want to scaffold another CRD
```

Comment:
couple of options here:
 - Update the kustomization file to include new resource file (for majority of the cases, this should be an easy update because users are not going to update this kustomization) ?
 - Create separate directory for each kind and include them as bases in the parent dir ?